base/src/model.rs

```diff
@@ -1172,6 +1172,18 @@ impl Model {
         }))
     }
 
+    /// Returns a boolean representing whether the cell contains a formula or not.
+    pub fn cell_contains_formula(&self, sheet_index: u32, row: i32, column: i32) -> bool {
```

Comment:
I would prefer if you can add the example without having to add a method in the public API. This is something I would like to discuss with you, but I would like the public API to be as minimal as possible.